scripts/setup-localstack.sh

```diff
@@ -8,7 +8,7 @@ awslocal lambda create-function \
   --function-name te-lambda \
   --zip-file fileb://dist/lambda.zip \
   --handler ds-caselaw-ingester/lambda_function.handler \
-  --runtime python3.11 \
+  --runtime python3.12 \
```

Comment:
Does this work with it using the API client, which uses saxonche, which is fussy about Python versions > 3.11?

Reply:
Dropped this runtime change, we can do it in another PR so it's more contained.